scripts/ui_release.sh

```diff
@@ -74,13 +77,81 @@ function bumpVersion() {
   fi
   # upgrade the @perses-dev/* dependencies on all packages
   for workspace in ${workspaces}; do
-    sed -E -i "" "s|(\"@perses-dev/.+\": )\".+\"|\1\"\^${version}\"|" "${workspace}"/package.json
+    # sed -i syntax is different on mac and linux
+    if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
I ran into this trying to automate this in github. Previously, I think it was only run on people's machines.

Reply:
Oh wow, TIL!